styles.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  height: 5rem;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 999;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10%;
+}
+
+.nav-links ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  list-style-type: none;
+  gap: 2.5rem;
+}
+
+.nav-links a {
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.logo a {
+  text-decoration: none;
+}
+
+.logo-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.25rem;
+  line-height: 1.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.main {
+  background-image: url("assets/bg-mobile.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 718px;
+  width: 100vw;
+  padding: 5rem 10%;
+  border-bottom-left-radius: 30%;
+}
+
+.main h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  color: #091e42;
+}
```

Comment:
- [Optional] Kindly increase the font size of your main title so it matches the figma design better. i believe a font size of 40px will do the trick
- [Optional] Kindly decrease the font size of your paragraph so it matches the figma design better. I believe a font size of 16px will do the trick
- [Optional] Kindly reduce the spacing above and below the let's connect text

Your design:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/37116322/227631577-a71ba1d2-7f12-4511-913b-edd01fdaa747.png">

Figma design:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/37116322/227631663-a3195323-fdcd-4e8e-a970-2a8656bbbdfb.png">